### DISCUSSION TITLE
Restrict foraging state at night

### DISCUSSION THREAD

**Discussion by Fiorella2891** [bmcclintock/momentuHMM#127] [Q&A]
Hello, I hope you are well.
I am writing to you because I am trying to fit an HMM to my seabird GPS data using the MomentumHMM package and among others identify the foraging state. This specie of bird does not dive at night, so I would like to insert a binomial covariate to restrict the foraging state to be identified only during daylight hours. I have been going through the manual but I still can't figure out how to do it. I would appreciate your help

Thank you very much for your help.

**Comment by bmcclintock** [comment]
If you only have two states, then the `knownStates` argument could be used to restrict non-daylight hours to the non-foraging state. It's a little bit more complicated with >2 states, but the restriction can still be imposed using the `fixPar` argument and including the binary covariate on the state transition probabilities via the `formula` argument (and the `formulaDelta` argument for the initial state probabilities). See #68 for an example with three states.
